test/rotoscope_test.rb

```diff
@@ -317,6 +331,18 @@ def test_trace_uses_io_objects
     assert_frames_consistent contents
   end
 
+  def test_stop_trace_before_start_does_not_raise
+    rs = Rotoscope.new(@logfile)
+    rs.stop_trace
+  end
+
+  def test_gc_rotoscope_without_stop_trace
+    rs = Rotoscope.new(@logfile)
+    rs.start_trace
+    rs = nil
+    GC.start
+  end
```

Comment:
Makes sense, but would be good to detail that in the title of the test; `_does_not_crash` perhaps.